docs/docs/identity-providers/readme.md

```diff
@@ -16,10 +16,18 @@ In this guide we'll cover how to do the following for each identity provider:
 
 1. Set a **[Redirect URL](https://www.oauth.com/oauth2-servers/redirect-uris/)** pointing back to Pomerium. For example, `https://${authenticate_service_url}/oauth2/callback`.
 2. Generate a **[Client ID]** and **[Client Secret]**.
-3. Configure Pomerium to use the **[Client ID]** and **[Client Secret]** keys.
+3. Generate a **[Service Account]** for additional IdP Data.
+4. Configure Pomerium to use the **[Client ID]**, **[Client Secret]**, and **[Service Account]** keys.
```

Comment:
sorry, I overlapped with you a bit here. 

https://github.com/pomerium/pomerium/pull/997/files#diff-cd8d17c18a1abc71a2a67fcbec810232R20

Reply:
That's fine; I'll rebase when that is merged.